src/apps/explorer/ExplorerApp.tsx

```diff
@@ -144,7 +144,8 @@ export const ExplorerApp: React.FC = () => {
         <StateUpdaters />
         <Switch>
           <Route path="/mainnet" component={RedirectMainnet} />
-          <Route path={['/xdai', '/rinkeby', '/']} component={AppContent} />
+          <Route path="/xdai" component={RedirectXdai} />
+          <Route path={['/gnosischain', '/rinkeby', '/']} component={AppContent} />
```

Comment:
Got a recommendation from Martin to use `gc` instead of `gnosischain`.

Might not be the clearest as you said, but the UI will have the network switcher where the full name will be shown